library/old_protocol/serialize.cpp

```diff
@@ -68,15 +68,15 @@ int serialize_error_response(dnet_net_state *st, const dnet_cmd &cmd_in,
 	return 0;
 }
 
-int serialize_lookup_request(dnet_net_state *st, std::unique_ptr<n2_request> msg_in,
+int serialize_lookup_request(dnet_net_state *, std::unique_ptr<n2_request> msg_in,
                              std::unique_ptr<n2_serialized> &out_serialized) {
 	auto &msg = static_cast<lookup_request &>(*msg_in);
 
 	out_serialized.reset(new n2_serialized{ msg.cmd, {} });
 	return 0;
 }
 
-int serialize_lookup_response(dnet_net_state *st, std::unique_ptr<n2_message> msg_in,
+int serialize_lookup_response(dnet_net_state *, std::unique_ptr<n2_message> msg_in,
```

Comment:
`st` is used

Reply:
Final commit compiles normally. Maybe some bug while rebase in intermediate commits, I will check and fix.

Reply:
Fixed